netci.groovy

```diff
@@ -268,10 +269,17 @@ def static addTriggers(def job, def branch, def isPR, def architecture, def os,
                 // Pri one gets a push trigger, and only for release
                 if (architecture == 'x64') {
                     if (configuration == 'Release') {
-                        // We expect release jobs to be Windows, Ubuntu, CentOS or OSX
+                        // We expect release jobs to be Windows, or in the cross list
                         assert (os == 'Windows_NT') || (os in Constants.crossList)
-                        if (isFlowJob || os == 'Windows_NT') {
-                            Utilities.addGithubPushTrigger(job)
+                        if (!os in bidailyCrossList) {
+                            if (isFlowJob || os == 'Windows_NT') {
+                                Utilities.addGithubPushTrigger(job)
+                            }
+                        } 
+                        else {
+                            if (isFlowJob) {
+                                Utilities.addPeriodicTrigger(job, '0 0 0/12 1/1 * ? *')
```

Comment:
It's better to avoid a bunch of things launching all at once and instead spread them randomly throughout the day.  I think there is a way to do this.  I think its: